ground/gcs/gcs.pri

```diff
@@ -67,6 +67,7 @@ macx {
     GCS_DATA_BASENAME = Resources
     GCS_DOC_PATH     = $$GCS_DATA_PATH/doc
     GCS_BIN_PATH     = $$GCS_APP_PATH/$${GCS_APP_TARGET}.app/Contents/MacOS
+    QMAKE_MACOSX_DEPLOYMENT_TARGET=10.9

```

Comment:
that's what i was trying to established, it was required for C++11. I'm just somewhat surprised since I'd think the language you compile in wouldn't change the binaries that are output and where they can run. Perhaps it has requires some libraries that are otherwise not supported.